blueprints/screening_app/__init__.py

```diff
@@ -0,0 +1,420 @@
+from mindmeld import Application
+import screening_app.prediabetes as pd
+
+app = Application(__name__)
+
+
+@app.handle(intent='greet')
+def welcome(request, responder):
+    """
+    When the user begins the conversation with a greeting. Explain the system options.
+    """
+    responder.reply(pd.WELCOME_MSG)
+
+
+@app.handle(intent='exit')
```

Comment:
It looks like the distinction between the two `exit` intents is not necessary. You could merge `prediabetes_screening.exit` into `greeting.exit` and the NLP and DM as implemented should behave the same.